docs/admin/install/install.rst

```diff
@@ -4,7 +4,12 @@ Installing SecureDrop Workstation
 Copy the submission key
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-In order to decrypt submissions, your SecureDrop Workstation will need a copy of the secret key from your SecureDrop instance's SVS. To protect this key and preserve the air gap, you will need to connect the SVS USB to a Qubes VM with no network access, and copy it from there to ``dom0``. Note that you cannot directly copy and paste to the ``dom0`` VM from another VM - instead, follow the steps below to copy the file into ``dom0``:
+In order to decrypt submissions, your SecureDrop Workstation will need a copy of the secret key from your SecureDrop instance's SVS.
+
+.. note::
+   The copy of your SecureDrop submission secret key `cannot be protected with a passphrase <https://www.qubes-os.org/doc/split-gpg/#current-limitations>`_. To export a copy that does not require a passphrase, see :doc:`/admin/reference/removing_gpg_passphrase`.
```

Comment:
I think that doc is out-of-date, the limitation for us is not that it can't work with split-gpg, but with our provisioning script (also users would get prompted for said passphrase quite a bit). Could drop that, or replace it with a link to the securedrop server docs about provisioning the Submission Key that speak to passphrases.

Reply:
can i check my understanding real quick - the scenario in which a user would need to remove the passphrase from the submission key is when they decide to enter a passphrase here at steps 9 and 10? https://docs.securedrop.org/en/stable/admin/installation/generate_submission_key.html#create-the-key

and if so, should the language change in that doc too? from "it's okay to skip passphrase" to "do not enter a passphrase"?

Reply:
IMO the language there is OK - the next step also assumes it. But we could tweak it. I guess the tension there is that for the SVS it could work either way (and we have no visibility by default into how folks set them up), but for SDW it's now more of a requirement. Either way these docs would need to reflect the no-passphrase requirement.